networkx/algorithms/centrality/betweenness.py

```diff
@@ -320,12 +320,15 @@ def _accumulate_edges(betweenness, S, P, sigma, s):
     return betweenness
 
 
-def _rescale(betweenness, n, normalized, directed=False, k=None):
+def _rescale(betweenness, n, normalized, directed=False, k=None, endpoints=False):
     if normalized:
         if n <= 2:
             scale = None  # no normalization b=0 for all nodes
         else:
-            scale = 1.0 / ((n - 1) * (n - 2))
+            if endpoints:
+                scale = 1.0 / ((n) * (n - 1)) # If endpoints are included, change the scale factor to include these nodes
```

Comment:
Let's place the line `from __future__ import division` at the head of the file but after the module-level docstring (i.e. just above the line `from heapq import heappush, heappop`). See https://www.python.org/dev/peps/pep-0238/. I like to remove the `.0` in this case because it is an unnecessary artifact, but it's more of a matter of preference, so you don't have to do it if you prefer to minimize changes here.

The same comment applies to the line in the `else` branch as well.